arg_scripts/routing_args.py

```diff
@@ -9,6 +9,8 @@ def __init__(self):
         self.paths_matrix = []  # Matrix of potential paths for a single request
         self.mod_formats_matrix = []  # Modulation formats corresponding to each path in paths_matrix
         self.weights_list = []  # Keeping track of one weight of the path (Length, XT, Etc.)
+        self.connection_index = []
```

Comment:
Each of these added to an argument object should be commented like the above, so the user can easily identify what it is/does